chain/chain/src/store.rs

```diff
@@ -1890,8 +1890,10 @@ impl<'a> ChainStoreUpdate<'a> {
             // 3g. Delete from ColBlocksToCatchup
             store_update.delete(ColBlocksToCatchup, block_hash.as_ref());
             // 3i. Delete from ColBlockRefCount
-            self.chain_store.block_refcounts.cache_remove(&block_hash.clone().into());
-            store_update.delete(ColBlockRefCount, block_hash.as_ref());
+            if height > 0 {
```

Comment:
Should it be `genesis_height` instead of 0?

Reply:
Yes.